jsonrpc/src/content/params.json

```diff
@@ -43,6 +43,17 @@
       }
     }
   },
+  "ContentPairs": {
```

Comment:
`ContentItems`? imo (at least from python land) this is the usual term used to identify a key/value piar

Reply:
Personally ambivalent about `ContentPair` vs `ContentItem`.  For me, `ContentPair` feels more descriptive (`KeyValuePair` even more so), but if `ContentItem` is more conventional in a way that the rest of you appreciate, then I support the choice.

Reply:
I don't really mind one or the other either. I think what @njgheorghita refers to is e.g. the `items` iterator for dictionary in Python? But sure, that's rather language specific.

Reply:
I prefer ContentPair's as it is more descriptive, I updated the PR to ContentItems, because I think getting the PR in takes priority over small nits I might have